packages/settings-library/src/settings_library/utils_service.py

```diff
@@ -2,14 +2,12 @@
 
 
 """
-
-
 from pydantic.networks import AnyUrl
 
 from .basic_types import PortInt
 
-DEFAULT_AIOHTTP_PORT: PortInt = 8000
-DEFAULT_FASTAPI_PORT: PortInt = 8080
+DEFAULT_AIOHTTP_PORT: PortInt = 8080
+DEFAULT_FASTAPI_PORT: PortInt = 8000
```

Comment:
minor: actually... we could use the same port for both apps.

Reply:
No problem. But I would suggest a separate PR :-)

Reply:
#2759